consonance-webservice/src/main/java/io/swagger/wes/api/ApiOriginFilter.java

```diff
@@ -0,0 +1,22 @@
+package io.swagger.wes.api;
+
+import java.io.IOException;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletResponse;
+
+@javax.annotation.Generated(value = "io.swagger.codegen.languages.JavaJerseyServerCodegen", date = "2017-09-15T17:06:31.319-07:00")
```

Comment:
Potentially helpful tip, it's actually possible to turn off data generation which can help avoid a bunch of commits in the future that are just the date changing.